modules/installation-user-infra-machines-advanced.adoc

```diff
@@ -176,127 +169,113 @@ $ ls $HOME/clusterconfig/
 auth  bootstrap.ign  master.ign  metadata.json  worker.ign
 ----
 
-At this point, you can use the Ignition config files as input to the ISO or PXE bare
+Now you can use the Ignition config files as input to the ISO or PXE bare
 metal installation procedures to install {op-system-first} systems.
 
 [id="installation-user-infra-machines-advanced_retaindisk_{context}"]
 === Retaining existing partitions
-For an ISO installation, you can add options to the `coreos-installer`
+For an ISO installation, you can add options to the `coreos-installer` command line
 that causes the installer to maintain one or more existing partitions.
-For a PXE installation, you can APPEND coreos.inst options to preserve partitions.
+For a PXE installation, you can `APPEND` `coreos.inst` options to preserve partitions.
 
 Saved partitions might be partitions from an existing {product-title}
-system that has data partitions that you want to keep or partitions
-that you just manually created. Here are a few tips:
+system that has data partitions that you want to keep. Here are a few tips:
 
-* Make sure you assign at least the recommended amount of disk space to the
-OpenShift partitions at the beginning of the disk.
+* Keep in mind that if you are saving existing partitions, those partitions will either leave enough space for {op-system}, or they will not and the installation will fail.
```

Comment:
What about: "If you save existing partitions, and those partitions do not leave enough space for {op-system}, installation will fail without damaging the saved partitions."